.gitignore

```diff
@@ -236,6 +236,7 @@ virtualenv-15.2.0-py2.py3-none-any.whl
 *.whl
 
 user.mk
+.xprocess/
```

Comment:
How about add removing `.hypothesis/`, `.xprocess/`, `.pytest_cache/` directory when we run `make clean`?

Reply:
Thank you
Changed:
- Add above folders to `.gitignore`
- Add `$ make cleanf-cache` to remove them (Added this keyword to `$ make clean`